app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/network/VpnNetworkStack.kt

```diff
@@ -68,15 +68,16 @@ interface VpnNetworkStack {
      * @param addresses the address you wish to set to the VPN service. They key contains the InetAddress of the address and
      * value should be the mask width.
      * @param dns the additional dns servers you wish to add to the VPN service
-     * @param routes the additional routes you wish to add to the VPN service. The key contains the InetAddress of the route and
-     * value should be the mask width.
+     * @param routes the routes (if any) you wish to add to the VPN service.
+     * The Pair<String, Int> contains the String IP address and the Int mask.
+     * If no routes are returned, the VPN will apply its own defaults.
      * @param appExclusionList the list of apps you wish to exclude from the VPN tunnel
      */
     data class VpnTunnelConfig(
         val mtu: Int,
         val addresses: Map<InetAddress, Int>,
         val dns: Set<InetAddress>,
-        val routes: Map<InetAddress, Int>,
+        val routes: List<Pair<String, Int>>,
```

Comment:
Why is this necessary? Not have Map<String, Int> instead?

Reply:
> Why is this necessary? Not have Map<String, Int> instead?

Yeah, on a second thought maybe a `map` is better. Made like that in case we had the same IP address and different mask. But I don't think that a realistic case